admin/updates/20160621-non-mbid-submission.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
+
+CREATE TYPE id_type AS ENUM ('mbid', 'msid');
+ALTER TABLE "lowlevel" RENAME COLUMN mbid TO gid;
+ALTER TABLE "lowlevel" ADD COLUMN is_mbid id_type;
```

Comment:
I would call gid_tipe because if we call it id_type it can be confusing and one can think that it is referenced to the lowlevel.id column while it is related to to lowlevel.gid column. Maybe we can change the name of the enum and call it git_type_enum.